### PR TITLE
Add bf16 matmul llvm conversion

### PR DESCRIPTION
**Pull request by jsetoain**

### CODE REVIEW

**Review by jgmelber — COMMENTED** [review]
Cool!

File: include/aie/Dialect/AIEVec/IR/AIEVecOps.td [hunk 1]
```diff
@@ -727,4 +728,27 @@ def AIEVec_ExtElemOp:
   let assemblyFormat = "$source `,` $index attr-dict `:` type($source) `,` type($index) `,` type($result)";
   let hasVerifier = 0;
 }
+
```
Comment (jamestcl-amd) [inline include/aie/Dialect/AIEVec/IR/AIEVecOps.td:731]: There are overlapping between this PR and the https://github.com/Xilinx/mlir-aie/pull/684. Just curious, do we intend to merge both PRs?

Comment (jsetoain) [reply include/aie/Dialect/AIEVec/IR/AIEVecOps.td:731]: Good catch! It shouldn't be. In my local repository this branch hangs from the other, and the PR includes both by default (my bad).

**Review by david-vc — COMMENTED** [review]
Hi @jsetoain : Is the PR still valid? Are you still seeking review of this?

**Comment by jsetoain** [comment]
> Hi @jsetoain : Is the PR still valid? Are you still seeking review of this?

No, and it's outdated and broken (I've got a newer version waiting for the dependency to land), I should have closed it long ago. Apologies.

**Pull request closed without merging**